### ISSUE TITLE
Add "featured pages" into docs homepage

### ISSUE THREAD

**Issue by IlCallo** [dreamonkey/quasar#29]
**Description**
_Clearly and concisely explain the problem.
You can mention unclear or missing information, typos, grammatical mistakes, etc.
Any contributions and suggestions are welcome!_

Shall we add some "featured pages" for pages which we want the users to discover, into the docs homepage?
We have most used pages, but some others like quasar utils and app extensions are often neglected as they are the last ones, but they are pretty important to know in the day-to-day dev experience

**Website Section URL**
_[Website live preview](https://quasar.beget.it), paste the page URL or section name (eg. Docs, if it happens on all docs page) below._

https://quasar.beget.it/docs

**Comment by IlCallo** [comment]
Calson opinion on this from Zeplin:

> I definitely agree with you about showing off some of those pages that are often ignored because users don't know they exist. We could select some particular pages from quasar utils and app extensions like date utils, formatter utils, ...

**Comment by IlCallo** [comment]
@yusufkandemir can you write here which pages are the most overlooked/underrated?

**Comment by IlCallo** [comment]
@hawkeye64 can you provide a list of UI AE we want to showcase in the homepage?

**Comment by yusufkandemir** [comment]
@IlCallo 
- **`quasar.conf.js`**: It is a bit hard to find this using global search because it's not a proper word. Also, there are lots of options but fewer sections, _which might indicate the page can be updated too_, so it is also hard to find individual options. It is naturally a commonly used page because `quasar.conf.js` is basically the heart of an app.
- **Boot Files**: Installing Vue plugins, creating some initialization logic, etc. is really common. But, people generally can't find it and ask on our channels, or they try to hack around with store, router, `App.vue`, etc.(_which is not always possible_) when they could use boot files instead.
- **Date utils**: This is really common in apps, people don't expect it to exist in Quasar, so they don't search or notice it. I helped a lot of people notice it, they then liked it and started using it.
- **Other utils**: `exportFile`, `debounce`, `throttle`, `uid`, etc. are cool, useful, and common. Often used from `lodash`. It would be better if we find a better name than `Other utils`.
- **Flexbox**(_Introduction to Flexbox_): Flex is really common. Maybe not really overlooked because it's used a lot in the examples, but still a good option to put in there.
- **Animations**: Similar to flexbox, not a crucial option, but I've found some people trying to integrate animate.css manually, or do some other stuff.
